fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java

```diff
@@ -762,15 +786,24 @@ private void writeFreezePane(Writer w) throws IOException {
      * @param w Output writer.
      * @param r Zero-based row number.
      * @param isHidden Whether or not this row is hidden
+     * @param customHeight Whether custom row height is set
+     * @param rowHeight Row height value in points to be set if customHeight is true
      * @param row Cells in the row.
      * @throws IOException If an I/O error occurs.
      */
-    private static void writeRow(Writer w, int r, boolean isHidden, Cell... row) throws IOException {
-        w.append("<row r=\"").append(r + 1);
+    private static void writeRow(Writer w, int r, boolean isHidden,
+    		boolean customHeight, double rowHeight, Cell... row) throws IOException {
```

Comment:
You could pass a single parameter of type `Double` or `Optional<Double>` for `rowHeight`?

Reply:
done